Updates: CSS Multicol guides (#27058)
mdn/content: files/en-us/web/css/css_multicol_layout/handling_content_breaks_in_multicol_layout/index.md

{{CSSRef}}

Content between column boxes in multi-column layout breaks in the same way that it is broken between pages in paged media. In both contexts, we control where and how things break by using properties of the CSS Fragmentation specification. In this guide, we see how fragmentation works in a _multi-column container_ or _multicol container_ for short.

## Fragmentation basics

> While the [CSS fragmentation](/en-US/docs/Web/CSS/CSS_fragmentation/) module provides details on how content breaks between the fragmentation containers or _fragmentainers_, as in multicol layout, the column box is a fragment container, the [multi-column layout](/en-US/docs/Web/CSS/CSS_multicol_layout/) module defines the {{cssxref("break-after")}}, {{cssxref("break-before")}}, and {{cssxref("break-inside")}} properties that provide some control within and between columns.

Reviewer: splitting the long sentence

```suggestion
The CSS fragmentation module provides details on how content breaks between the fragmentation containers or _fragmentainers_. The [multi-column layout](/en-US/docs/Web/CSS/CSS_multicol_layout) module, on the other hand, defines the {{cssxref("break-after")}}, {{cssxref("break-before")}}, and {{cssxref("break-inside")}} properties that provide some control within and between columns. In multicol layout, a column box is a fragment container.
```

## Breaks between lines

> The {{cssxref("orphans")}} and {{cssxref("widows")}} properties, part of the [CSS fragmentation](/en-US/docs/Web/CSS/CSS_fragmentation/) module, are also useful and worth mentioning. The `orphans` property controls the number of lines left on their own at the end of a fragment. The `widows` property controls the number left on their own at the start of a fragment.

Reviewer: ```suggestion
The {{cssxref("orphans")}} and {{cssxref("widows")}} properties, part of the CSS fragmentation module, are also useful and worth mentioning. The `orphans` property controls the number of lines left on their own at the end of a fragment. The `widows` property controls the number left on their own at the start of a fragment.
```
